.github/workflows/resources/docker-compose.postgres.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  postgres:
+    # NOTE: Support for PostgreSQL 13 ends on Nov 2025, bump to 14 after that
+    # https://endoflife.date/postgresql
+    image: postgres:13
```

Comment:
Should we keep this in sync with what we use with AWS RDS, which is currently Postgres v15?

Reply:
Ah wait, wrong repo 😅